README.adoc

```diff
@@ -48,7 +48,6 @@ Build Polkadot PoC-4 by cloning this repository and running the following comman
 ```bash
 git checkout v0.4
 ./scripts/init.sh
-./scripts/build.sh
```

Comment:
I think this one needs to be kept since we're instructing to checkout `v0.4` branch.